Composer/packages/server/src/services/telemetry.ts

```diff
@@ -59,11 +53,6 @@ if (instrumentationKey) {
 
     const data = envelope.data as AppInsights.Contracts.Data<AppInsights.Contracts.RequestData>;
 
-    if (alwaysTrackEvents.includes(data.baseData.name as TelemetryEventName) && !telemetry?.allowDataCollection) {
```

Comment:
Where is the PII being stripped?

Reply:
I'll double-check this. I don't actually see it offhand, so I'll make sure it gets done somewhere along the line.